docs/platforms/javascript/common/user-feedback/configuration/index.mdx

```diff
@@ -8,43 +8,48 @@ description: "Learn about the general User Feedback configuration fields."
+The following config options apply to version 8.0.0 of the JavaScript SDK. If you are using any version 7 of the SDK refer to [configuration options](/platforms/javascript/user-feedback/configuration/v7/) for that version of the SDK.
 
 ### General
 
 The following options can be configured for the integration in `feedbackIntegration({})`:
 
-| key            | type                                       | default    | description                                                                                                                                                                                                                              |
-| -------------- | ------------------------------------------ | ---------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `autoInject`   | `boolean`                                  | `true`     | Injects the Feedback widget into the application when the integration is added. Set `autoInject: false` if you want to call `feedback.attachTo()` or `feedback.openDialog()` directly, or only want to show the widget on certain views. |
-| `showBranding` | `boolean`                                  | `true`     | Displays the Sentry logo inside of the form                                                                                                                                                                                              |
-| `colorScheme`  | <code>"system" \| "light" \| "dark"</code> | `"system"` | The color theme to use. `"system"` will follow your OS color scheme.                                                                                                                                                                     |
+| Key            | Type                                       | Default           | Description                                                                                                                                                                                                                                |
+| -------------- | ------------------------------------------ | ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `autoInject`   | `boolean`                                  | `true`            | Injects the Feedback widget into the application when the integration is added. Set `autoInject: false` if you want to call `feedback.attachTo()` or `feedback.createWidget()` directly, or only want to show the widget on certain views. |
```

Comment:
In the example below you use `createForm`, not sure which is correct

Reply:
createWidget returns the Actor, so i think that's correct here. 

The example below should've used createForm, which is async and just does the form. I also added a quick example near the top under the heading `Auto-Inject vs. Manual Injection` which uses createWidget

So the examples down the page are:
`createWidget` which is at the top, near the notes about `autoInject`

Under byo-button we've got `createForm`

under byo-w we've got `sendFeedback` itself.